src/components/text-area/text-area.tsx

```diff
@@ -337,13 +332,13 @@ export class TextArea
 
   labelEl: HTMLCalciteLabelElement;
 
-  textareaEl: HTMLTextAreaElement;
+  textAreaEl: HTMLTextAreaElement;
 
   footerEl: HTMLElement;
 
-  endSlotHasElements: boolean;
+  @State() endSlotHasElements: boolean;
```

Comment:
Can you make sure component props/variables in alphabetical order? I think we can keep variables w/o decorators separate from the ones that use them.